pkg/pool-linear/contracts/LinearPoolRebalancer.sol

```diff
@@ -146,7 +150,11 @@ abstract contract LinearPoolRebalancer {
         return reward;
     }
 
-    function _rebalanceExcessOfMainToken(uint256 excessMainAmount, address recipient) private returns (uint256) {
```

Comment:
@belbix Please take a look. I've changed base class visibility to be able to change this method in our implementation. Maybe you have better idea?